### PR TITLE
[12.0][IMP] base_tier_validation_formula

### PR DESCRIPTION
**Pull request by ps-tubtim**
From issue: https://github.com/OCA/server-ux/issues/114

- Add **Domain & Formula** in `definition_type`.

### CODE REVIEW

**Comment by ps-tubtim** [comment]
@LoisRForgeFlow @kittiu Please Review.

File: base_tier_validation_formula/__init__.py [hunk 1]
```diff
@@ -1 +1,3 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (LoisRForgeFlow) [inline base_tier_validation_formula/__init__.py]: Please don't add licenses in init files, it only generates bigger (and unneeded) diffs.

Comment (ps-tubtim) [reply base_tier_validation_formula/__init__.py]: Removed

**Review by kittiu — APPROVED** [review]
Add some readme for new feature,

---

To define the domain,
* By python code choose the Formula option in the Definition field.
* By both domain and python code, choose the Domain & Formula option in the Definition field.

To define the reviewers by python code choose Python Expression option in the Validated by field.

**Comment by ps-tubtim** [comment]
@kittiu README changed.

**Comment by ps-tubtim** [comment]
> Add some readme for new feature,
> 
> To define the domain,
> 
> * By python code choose the Formula option in the Definition field.
> * By both domain and python code, choose the Domain & Formula option in the Definition field.
> 
> To define the reviewers by python code choose Python Expression option in the Validated by field.

@kittiu README changed.

**Review by dreispt — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
@ps-tubtim could you forward port to v13?

**Comment by ps-tubtim** [comment]
@LoisRForgeFlow I did it. https://github.com/OCA/server-ux/pull/132

**Pull request merged**